src/stories/components/svg/EndgameAtlasBudgets.tsx

```diff
@@ -0,0 +1,84 @@
+import * as React from 'react';
+
+interface Props {
+  width?: number;
+  height?: number;
+  style?: React.CSSProperties;
```

Comment:
why not use `className` like the other icons?

Reply:
you are right